cbpi/extension/mashstep/__init__.py

```diff
@@ -34,7 +33,7 @@ async def on_stop(self):
 
     async def reset(self):
         self.summary = ""
-        self.timer = Timer(int(self.props.Timer) *60 ,on_update=self.on_timer_update, on_done=self.on_timer_done)
+        self.timer = Timer(int(self.props.Timer) * 60, on_update=self.on_timer_update, on_done=self.on_timer_done)
 
     async def run(self):
         while True:
```

Comment:
This should now also be changed to `while self.running == True:` because the runnig flag now is set and used to control the run method.